index.js

```diff
@@ -189,10 +191,42 @@ class Emittery {
 		};
 	}
 
-	constructor() {
+	static get isDebug() {
```

Comment:
```suggestion
	static get isDebugEnabled() {
```

Just to make it clear.